paddle/framework/framework.proto

```diff
@@ -40,8 +40,8 @@ message OpDesc {
   };
 
   message Var {
-    required string name;                 // e.g. "X"
-    optional int dup = 2 [ default = 0 ]; // e.g., "1"
+    required string op_proto_name = 1;
```

Comment:
It is not `global`/`local` relationship.
The relationship of them should be similar with function's parameter and argument.
Like,

```python
fc_layer(X=["x1", "x2"], W=["w1", "w2"])
```

`X` is op_proto_name, `[x1, x2]` is var_names.